vnext/Folly/Folly.vcxproj

```diff
@@ -278,16 +278,19 @@
     <Error Condition="!Exists('$(SolutionDir)packages\boost.1.76.0.0\build\boost.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)packages\boost.1.76.0.0\build\boost.targets'))" />
   </Target>
   <Target Name="DownloadFolly" BeforeTargets="PrepareForBuild">
-    <Message Importance="High" Text="Downloading folly..." Condition="!Exists('$(FollyDir)..\.follyzip\folly-$(FollyVersion).zip')" />
-    <DownloadFile Condition="!Exists('$(FollyDir)..\.follyzip\folly-$(FollyVersion).zip')"
+    <PropertyGroup>
+      <FollyZipDir>$(IntDir)\.follyzip</FollyZipDir>
```

Comment:
is this guaranteed to be set for customer projects (both C++ and C#)?

Reply:
also need to make sure this works for non-CLI projects

Reply:
e.g. if IntDir isn't set, use the old path

Reply:
See https://github.com/microsoft/react-native-windows/blob/main/vnext/Directory.Build.props

Reply:
Yes, IntDir is the standard defined intermediate output dir. See https://docs.microsoft.com/en-us/cpp/build/reference/common-macros-for-build-commands-and-properties?view=msvc-160